src/main/java/com/novi/TechItEasy/exceptions/RecordNotFoundHandler.java

```diff
@@ -0,0 +1,21 @@
+package com.novi.TechItEasy.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class RecordNotFoundHandler {
+    @ExceptionHandler(value = RecordNotFoundException.class)
+    public ResponseEntity<Object> handleRecordnotFoundException(RecordNotFoundException e){
+// 1.create payload containing  exception  details;
+        HttpStatus notFound = HttpStatus.NOT_FOUND;
```

Comment:
Waarom zet je dit niet rechtstreeks in de return?

Reply:
zodat ik kan customized message kan thow'en in the return. maar juist dat ik exception controller heb gebruikt alleen in Television Services en niet overal.